source/FreeRTOS_TCP_Reception.c

```diff
@@ -194,10 +169,10 @@
  *         negative value wherein the calling function should not process this packet any
  *         further and drop it.
  */
-    static int32_t prvSingleStepTCPHeaderOptions( const uint8_t * const pucPtr,
-                                                  size_t uxTotalLength,
-                                                  FreeRTOS_Socket_t * const pxSocket,
-                                                  BaseType_t xHasSYNFlag )
+    int32_t prvSingleStepTCPHeaderOptions( const uint8_t * const pucPtr,
```

Comment:
Why did we remove the static qualifier? Are these not used in this file only?

I think we should keep the static qualifier and remove the declaration from the header.

Reply:
These two were added to the header file for running unittest.